web/tests/functional/report_viewer_api/test_get_run_results.py

```diff
@@ -447,3 +447,20 @@ def test_get_checker_labels(self):
         self.assertEqual(len(checker_labels), 2)
         self.assertEqual(set(checker_labels[0]), div_zero_labels)
         self.assertEqual(set(checker_labels[1]), set())
+
+    def test_report_blame_info(self):
+        """
+        Get run results and check that blame info exists
+        when get_details is set.
+        """
+        runid = self._runid
+        simple_filter = ReportFilter()
+        run_results = self._cc_client.getRunResults([runid],
+                                                    100,
+                                                    0,
+                                                    None,
+                                                    simple_filter,
+                                                    None,
+                                                    True)
+
+        self.assertTrue(any(res.blameInfo for res in run_results))
```

Comment:
Can we test the json output as well? At least that is doesn't crash.

Reply:
I think it is sufficient to check only the existence of the blame info. If someone modified the test files, the blame info whould be also changed, which whould break the test.